.github/workflows/pull_request.yml

```diff
@@ -25,6 +30,12 @@ jobs:
         run: npm ci
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
+      - name: Pull Latest docker image
+        run: docker pull amazon/dynamodb-local:latest
+      - name: Create test credentials
+        run: |
+          mkdir $HOME/.aws
+          echo -e "[default]\naws_access_key_id=fakeTestString\naws_secret_access_key=fakeTestString\naws_session_token=fakeTestString" > $HOME/.aws/credentials
```

Comment:
Is this now required for local development too? If so, why?

Reply:
This can probably be refined. The client that runs in the ddb local plugin needs any credentials, otherwise it throws an error: `CredentialsProviderError: Could not load credentials from any providers`. I have an update for that package that should fix that though, working on it now, so this wont be needed

Reply:
Actually this is needed, or at least the env variables (which I can try setting differently) are needed. The `@tables` pragma does some stuff, and since this is pointing to the local ddb instead of dynalite it needs credentials in the _createTables function in the sandbox code.

Reply:
Do your recent changers to the plugin eliminate the need for this?